charts/app-of-apps/values.yaml

```diff
@@ -4,3 +4,10 @@ ckanHelmValues:
 
   monitoring:
     enabled: true
+
+datagovukHelmValues:
+  publish:
+    count: 1
```

Comment:
Is `count` the number of replicas? (I'm guessing so, but I'd have to read through the templates to be certain — which hurts maintainability.) If so, best use the conventional name `replicaCount`. (similarly throughout)

Reply:
ok I'll update